### ISSUE TITLE
Aroma version?

### ISSUE THREAD

**Issue by SwirlyStone5877** [dimok789/homebrew_launcher#72]
Sorry for past me.

**Comment by hazbeans** [comment]
if u have tiramisu reboot 2 tiramisu, if not, womp womp

**Comment by DraconicNEO** [comment]
Agreed, Aroma literally has support for homebrew_launcher.wuhb and hiding all other homebrew from the menu. I'm not sure why it hasn't been taken advantage of.

**Comment by Masamune3210** [comment]
What's the point of having the homebrew launcher as a wuhb if Aroma has support for showing homebrew on the Home Menu? If the intent is running things under the homebrew channel that won't run from Aroma's system, bringing the homebrew channel back isn't going to solve that, the method isn't compatible with Aroma.

**Comment by DraconicNEO** [comment]
> What's the point of having the homebrew launcher as a wuhb if Aroma has support for showing homebrew on the Home Menu? If the intent is running things under the homebrew channel that won't run from Aroma's system, bringing the homebrew channel back isn't going to solve that, the method isn't compatible with Aroma.

I don't think you understand the point isn't to run things you wouldn't run, the point is that some people don't want to clutter their menu with many rpx homebrews that they'll use once or seldomly. Is this misunderstanding the core reason this feature is still unused to this day? 

That's literally what this very criminally unused feature is for:
```
    - Hides all homebrew from the Wii U Menu except the `sd:/wiiu/apps/homebrew_launcher.wuhb` and `sd:/wiiu/apps/homebrew_launcher/homebrew_launcher.wuhb`
```

**Comment by DraconicNEO** [comment]
> Sorry for past me.

Don't apologize, this is a good idea and there is good reason to support it or Aroma wouldn't have the option.
